docs/tutorials/installation/0-basic-software-installation.md

```diff
@@ -165,27 +165,38 @@ Create or update the conda environments with :
 $ ${FEDBIOMED_DIR}/scripts/configure_conda
 ```
 
-List the existing conda environments and check the 3 environments `fedbiomed-network` `fedbiomed-node` `fedbiomed-researcher` were created :
+List the existing conda environments and check the 3 environments `fedbiomed-gui` `fedbiomed-node` `fedbiomed-researcher` were created :
 
 ```
 $ conda env list
 [...]
-fedbiomed-network        /home/mylogin/.conda/envs/fedbiomed-network
+fedbiomed-network        /home/mylogin/.conda/envs/fedbiomed-gui
 fedbiomed-node           /home/mylogin/.conda/envs/fedbiomed-node
 fedbiomed-researcher     /home/mylogin/.conda/envs/fedbiomed-researcher
 [...]
 ```
 
+!!! note "Conda environment for Fed-BioMed Network Component"
+    The Fed-BioMed network component, which consists of two modules, namely `MQTT` and `Restful`, runs in a Docker container. Although `fedbiomed-network` exists as a conda environment, it is not directly used on the local system where Fed-BioMed is installed. However, it can be used to launch the `restful` module locally instead of using Docker.
+    
+    The conda environment can be install with the following command
```

Comment:
Hi @sharkovsky,
Thnaks. Your IDE might display as it is rendered as code but it isn't when the docs are built. And there is  indentation because it should be under note: "Conda environment for Fed-BioMed Network Component"

Reply:
ah shit, sorry I trusted the IDE too much!